src/main/java/org/elasticsearch/action/support/nodes/NodesOperationRequest.java

```diff
@@ -43,6 +43,11 @@ protected NodesOperationRequest() {
 
     }
 
+    protected NodesOperationRequest(ActionRequest request, String... nodesIds) {
```

Comment:
does that mean that all node operation requests suffer from the same issue? did you check for other potential bugs around this?